src/components/Input/inputRadio/radio.js

```diff
@@ -114,7 +114,6 @@ Radio.defaultProps = {
     disabled: false,
     onChange: () => {},
     tabIndex: undefined,
-    onClick: () => {},
```

Comment:
we should avoid changing the component API since there are uses that relies on this prop, we can't break it

Reply:
Sorry for that it was giving me a lint error and I supposed it was not needed. However I added a commit. Adding the prop back. Please review.